misc/registrar_validation/test.py

```diff
@@ -0,0 +1,364 @@
+# pylint: skip-file
```

Comment:
Never ever use "test" as a name for anything, unless it's really just a test of something to be deleted.  What does this actually _do_ for the "test"?  Take it one more level and add at least some description into the name!